lib/Parser/Parse.cpp

```diff
@@ -1703,11 +1703,10 @@ void Parser::BindPidRefsInScope(IdentPtr pid, Symbol *sym, int blockId, uint max
 
         if (ref->IsModuleExport())
         {
-            Assert(sym->GetIsGlobal());
```

Comment:
I assume this is no longer true because the exports are now considered local to the top-level function body?